source/docs/fancy_logger.md

```diff
@@ -1,38 +1,39 @@
-## Fancy Logger: Flexible Granularity Log Control in Envoy
+## Fine-Grain Logger: Flexible Granularity Log Control in Envoy
+### **Note**: there is a name conversion, and previous is **Fancy** Logger
```

Comment:
Note: this was renamed from Fancy Logger as the name was opaque and didn't reflect its fine grain logging capabilities.

Reply:
Thanks!